<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Matlibplot</class>
 <widget class="QMainWindow" name="Matlibplot">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>808</width>
    <height>650</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>650</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>808</width>
    <height>650</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CS-RC5 Control Panel</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>img/playstore.png</normaloff>img/playstore.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#2E3440;
color:#D8DEE9;


</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>800</width>
     <height>640</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>800</width>
     <height>640</height>
    </size>
   </property>
   <property name="font">
    <font>
     <kerning>true</kerning>
    </font>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <property name="leftMargin">
     <number>9</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="styleSheet">
       <string notr="true">border:0px solid white;</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="0">
        <widget class="matlibplot1" name="widget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 1px solid white;</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QFrame" name="frame">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>770</width>
           <height>80</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>770</width>
           <height>85</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QFrame{
border-bottom: 1px solid #404859;
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QLabel" name="label_10">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>40</y>
            <width>181</width>
            <height>14</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>14</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>13</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">border-bottom:0px solid;</string>
          </property>
          <property name="text">
           <string>Chamber status: Closed</string>
          </property>
         </widget>
         <widget class="QSlider" name="Timer">
          <property name="geometry">
           <rect>
            <x>229</x>
            <y>40</y>
            <width>321</width>
            <height>22</height>
           </rect>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="mouseTracking">
           <bool>true</bool>
          </property>
          <property name="tabletTracking">
           <bool>true</bool>
          </property>
          <property name="focusPolicy">
           <enum>Qt::ClickFocus</enum>
          </property>
          <property name="acceptDrops">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">QSlider::groove:horizontal {
    border: 1px solid #999999;
	border-radius: 3px;
    height: 8px; /* the groove expands to the size of the slider by default. by giving it a height, it has a fixed size */
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #B1B1B1, stop:1 #c4c4c4);
    margin: 2px 0;
}

QSlider::handle:horizontal {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #b4b4b4, stop:1 #8f8f8f);
    border: 1px solid #5c5c5c;
    width: 18px;
    margin: -2px 0; /* handle is placed by default on the contents rect of the groove. Expand outside the groove */
    border-radius: 3px;
}</string>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>10</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksAbove</enum>
          </property>
         </widget>
         <widget class="QLabel" name="label_11">
          <property name="geometry">
           <rect>
            <x>230</x>
            <y>20</y>
            <width>16</width>
            <height>16</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">border-bottom:0px solid white;</string>
          </property>
          <property name="text">
           <string>1</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_12">
          <property name="geometry">
           <rect>
            <x>540</x>
            <y>20</y>
            <width>21</width>
            <height>16</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">border-bottom: 0px solid white;</string>
          </property>
          <property name="text">
           <string>10</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_13">
          <property name="geometry">
           <rect>
            <x>310</x>
            <y>0</y>
            <width>191</width>
            <height>20</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">border-bottom: 0px solid white;</string>
          </property>
          <property name="text">
           <string>Adjust data collection rate (sec)</string>
          </property>
         </widget>
         <widget class="QPushButton" name="SavedFileButton">
          <property name="geometry">
           <rect>
            <x>600</x>
            <y>40</y>
            <width>131</width>
            <height>23</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">border:1px solid  rgb(199, 199, 199);
border-radius: 4px;
padding-top: 2px;
padding-bottom:2px;
padding-left:10px;
padding-right:10px;</string>
          </property>
          <property name="text">
           <string>View Saved File</string>
          </property>
         </widget>
         <widget class="QFrame" name="frame_2">
          <property name="geometry">
           <rect>
            <x>70</x>
            <y>10</y>
            <width>57</width>
            <height>24</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true"> background: #677590;
border-bottom:0px solid white;
border-radius:10px;</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <widget class="QSlider" name="toggleButton">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>1</y>
             <width>54</width>
             <height>22</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <strikeout>false</strikeout>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="mouseTracking">
            <bool>true</bool>
           </property>
           <property name="toolTipDuration">
            <number>0</number>
           </property>
           <property name="styleSheet">
            <string notr="true">QSlider::groove:horizontal {
	width: 50px;
	border-bottom: 0px solid white;
	border-radius:10px;
    height: 20px; /* the groove expands to the size of the slider by default. by giving it a height, it has a fixed size */
    background: #677590;
    margin: 0px 0px;

}

QSlider::handle:horizontal {
	width:20px;
    height: 20px;
	border-radius: 10px;
    background: #2E3440;
	 margin: 0px 4px; 
   /* expand outside the groove */
}
</string>
           </property>
           <property name="maximum">
            <number>1</number>
           </property>
           <property name="sliderPosition">
            <number>0</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="tickPosition">
            <enum>QSlider::NoTicks</enum>
           </property>
          </widget>
         </widget>
         <widget class="QPushButton" name="StopDLButton">
          <property name="geometry">
           <rect>
            <x>600</x>
            <y>10</y>
            <width>131</width>
            <height>23</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">border:1px solid  rgb(199, 199, 199);
border-radius: 4px;
padding-top: 2px;
padding-bottom:2px;
padding-left:10px;
padding-right:10px;</string>
          </property>
          <property name="text">
           <string>Stop Logging</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>matlibplot1</class>
   <extends>QWidget</extends>
   <header>matlibplot1.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
